lib/modules/swarm/upload.js

```diff
@@ -1,50 +1,41 @@
 require('colors');
 let async = require('async');
-let shelljs = require('shelljs');
 
 class Swarm {
   constructor(options) {
     this.options = options;
     this.buildDir = options.buildDir || 'dist/';
```

Comment:
add this.web3 and this.storageConfig

Reply:
correction/addition, it should  actually be in swarm/index.js (which is the actual module initialization), then the module can pass the params to upload

Reply:
Updated.